_extras/guide.md

```diff
@@ -19,7 +19,7 @@ is a great way to contribute.
 [Central processing unit](https://en.wikipedia.org/wiki/CPU)
 :     or simply *processor* is the (hardware) component of a computer that
       executes the instructions supplied by a program (software). Most modern
-      desktop computers have multi-core processors *e.g.* devices with 2 CPUs
+      desktop computers have multi-core processors e.g. devices with 2 CPUs
```

Comment:
Strictly speaking this is out of scope for the goal of this PR, but the sentence is a mild *non sequitur*. The first clause says that modern computers have multi-core processors, while the examples relate to translating the number of cores into their corresponding terms. 

At the very least, mechanically, there should be a comma before the "e.g."